code/src/Movie.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from 'react'
+import { useParams } from 'react-router-dom'
+
+
+import 'Movie.css'
+
+export const Movie = () => {
+  const { movieId } = useParams()
+  const [movies, setMovies] = useState([])
+  const [loading, setLoading] = useState([])
+
+  useEffect(() => {
+    setLoading(true)
+    fetch(`https://api.themoviedb.org/3/movie/${movieId}?api_key=2a89f9965a48e33d809fbd966cc6a018&language=en-US`)
+      .then((res) => res.json())
+      .then((json) => {
+        console.log(json)
+        setMovies(json)
+        setLoading(false)
+      })
+  }, [movieId])
+
+  if (loading === true) {
```

Comment:
This means true as well! False would be `!loading`
```suggestion
  if (loading) {
```